app/src/gunw.py

```diff
@@ -0,0 +1,85 @@
+import json
+import os
+import pathlib
+from dataclasses import dataclass
+
+import boto3
+
+import util
+
+
+@dataclass(frozen=True)
+class JobTypeConfig:
+    hyp3_urls: list[str] | None = None
+    user_id: str | None = None
+
+
+GUNW_USERNAME = 'access_cloud_based_insar'
+A19_URL = 'https://hyp3-a19-jpl.asf.alaska.edu'
+TIBET_URL = 'https://hyp3-tibet-jpl.asf.alaska.edu'
+
+
+JOB_TYPE_CONFIGS = {
+    'ARIA_S1_GUNW': JobTypeConfig(),
+    'INSAR_ISCE': JobTypeConfig(hyp3_urls=[A19_URL, TIBET_URL], user_id=GUNW_USERNAME),
+    'ARIA_RAIDER': JobTypeConfig(hyp3_urls=[A19_URL], user_id=GUNW_USERNAME),
+}
```

Comment:
The double None in the dataclass feels weird to me. I might do something like this (also did a bit of name changing):
 
```suggestion
@dataclass(frozen=True)
class JobTypeIngestConfig:
    hyp3_urls: list[str]
    user_id: str


GUNW_USERNAME = 'access_cloud_based_insar'
A19_URL = 'https://hyp3-a19-jpl.asf.alaska.edu'
TIBET_URL = 'https://hyp3-tibet-jpl.asf.alaska.edu'


INGEST_CONFIGS = {
    'ARIA_S1_GUNW': None,
    'INSAR_ISCE': JobTypeIngestConfig(hyp3_urls=[A19_URL, TIBET_URL], user_id=GUNW_USERNAME),
    'ARIA_RAIDER': JobTypeIngestConfig(hyp3_urls=[A19_URL], user_id=GUNW_USERNAME),
}
```

Reply:
I can see that, I think I was trying to leave open the possibility that we may want to filter by HyP3 URL or `user_id` but not both. E.g. I could imagine needing to set one of those parameters to `None` for either `INSAR_ISCE` or `ARIA_RAIDER` at some point. But idk for sure. I'll let this sit for now.